packages/plugin-react/src/index.d.ts

```diff
@@ -0,0 +1,31 @@
+import { Plugin } from 'vite'
+import { TransformOptions, ParserOptions } from '@babel/core'
+
+export interface Options {
+  include?: string | RegExp | Array<string | RegExp>
+  exclude?: string | RegExp | Array<string | RegExp>
+  /**
+   * Enable `react-refresh` integration. Vite disables this in prod env or build mode.
+   * @default true
+   */
+  fastRefresh?: boolean
+  /**
+   * Set this to `"automatic"` to use [vite-react-jsx](https://github.com/alloc/vite-react-jsx).
+   * @default "classic"
```

Comment:
Would it be good to set default as "automatic" ? As a breaking change for new plugin release will be more acceptable.